be/src/exec/pipeline/query_context.cpp

```diff
@@ -162,6 +162,15 @@ MemTracker* QueryContext::operator_mem_tracker(int32_t plan_node_id) {
     return mem_tracker.get();
 }
 
+Status QueryContext::init_spill_manager(const TQueryOptions& query_options) {
+    Status st;
+    std::call_once(_init_spill_manager_once, [this, &st, &query_options]() {
+        _spill_manager = std::make_unique<spill::QuerySpillManager>(_query_id);
```

Comment:
_spill_manager created in init_query_once why we create a new one?

Reply:
> _spill_manager created in init_query_once why we create a new one?

we need query_options when we init spill manager, I have removed it from init_query_once